assets/php/class-wgpb-block-library.php

```diff
@@ -378,6 +389,7 @@ public static function print_script_settings() {
 				'userLocale'    => get_user_locale(),
 				'weekdaysShort' => array_values( $wp_locale->weekday_abbrev ),
 			),
+			'billingFields' => $billing_fields,
```

Comment:
I like how the other PR is adding `wc_checkout_block_data` for this data - can this be added to that object instead of `wcSettings`?